anvilPoll2024MainAnalysis.Rmd

```diff
@@ -11,7 +11,7 @@ library(here)
 library(tidyverse)
 library(magrittr)
 library(grid) #for Grobs
-
+library(scales)
```

Comment:
any reason this is needed? I was able to render without it

Reply:
I use `scales` for the `pretty_breaks` function. When I test locally, I load an RDS instead of depending on knitting `TidyData.Rmd`, but when I've tried to remove `library(packagename)` from the extra analysis Rmd, it complains even if that package is loaded in the main analysis.

Reply:
But now I just used scales pretty_breaks in the extra analysis Rmd and didn't load it there and it did work!?

Reply:
Ok, we've got the package loading sorted! -- thank you!